src/content/docs/mlops/integrations/openai-integration.mdx

```diff
@@ -75,24 +82,144 @@ openai.Completion.create(
     temperature=0 
 )
 ```
+</Step>
+</Steps>
+ </Collapser>
+
+
+
+<Collapser 
+    id="install-with-node" 
+    title="Install with Node.js"
+  >
+<Steps>
+<Step>
+### Install OpenAI [#install-openai-node]
+
+Use your favorite package manager and run:
+
+```
+npm install @newrelic/openai-observability openai
+```
+</Step>
+<Step>
+### Getting started [#getting-started-node]
+
+Call `monitorOpenAI` with the `openAI`instance. 
+
+Here is some example code:
+
+```
+import { Configuration, OpenAIApi } from 'openai';
+import { monitorOpenAI } from '@newrelic/openai-observability';
+
+const configuration = new Configuration({
+  apiKey: 'OPENAI_API_KEY',
+});
+
+const openAIApi = new OpenAIApi(configuration);
+
+monitorOpenAI(openAIApi, {
+  applicationName: 'MyApp',
+  newRelicApiKey: 'NEW_RELIC_LICENSE_KEY',
+});
+
+const response = await openAIApi.createChatCompletion({
+  model: 'gpt-3.5-turbo',
+  messages: [{ role: 'user', content: 'What is Observability?' }],
+});
+
+console.log(response.data.choices[0].message);
+```
+
+Make sure that you add module type to package.json:
+
+```
+"type": "module"
+```
+</Step>
+
+<Step>
+### Initialization options [#modify-code]
```

Comment:
It looks like this is the part of the update that has the integration "run" when the app spins up? I think that's how SMEs use initialization. A little more context here would be helpful